### DISCUSSION TITLE
Disable async chunk loading

### DISCUSSION THREAD

**Discussion by balsanka** [webpack/webpack#15028] [Q&A]
How to disable async loading chunks?
I added static paths, so I don't need to load chunked js and css via script, it's just cause errors and braking script.

**Comment by alexander-akait** [comment]
You should not do it, most likely you do not understand something, why do you want to disable it? No need to load all files on initial opened page, it bad for your clients

**Reply by balsanka** [reply]
Found solution!
https://pocketsun.net/web-development/vuejs-webpack-loading-chunk-files-failed/
Sorry but it's my mistake, I forced to Grouping Components in the Same Chunk! :(
